src/CMSON.cc

```diff
@@ -0,0 +1,511 @@
+//
+//  CMSON.cc
+//  snowcrash
+//  C Implementation of MSON.h for binding purposes
+//
+//  Created by Pavan Kumar Sunkara on 07/01/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#include "CMSON.h"
+#include "snowcrash.h"
+
+/*----------------------------------------------------------------------*/
```

Comment:
Btw I am not insisting on these decorations but if you like them I do not care :)